ui/app/component/ec2/selfservice/Ec2SelfServiceController.js

```diff
@@ -98,6 +100,15 @@ class Ec2SelfServiceController extends GatekeeperSelfServiceController {
             disableBackgroundColor: 'rgba(0,0,0,0.12)'
         };
 
+        gkEc2ConfigService.fetch().then((response) =>{
+            let data = response.data;
+            vm.ticketIdFieldMessage = data.ticketIdFieldMessage;
+            vm.ticketIdFieldRequired = data.ticketIdFieldRequired;
+            vm.explanationFieldRequired = data.explanationFieldRequired;
+        }).catch(()=>{
+            throw new Error('Error fetching ticket systems');
```

Comment:
We should display a message on the UI indicating that it couldn't properly get the ticket properties instead of tossing a Javascript exception